frontend/src/component/p-photo-tiles.vue

```diff
@@ -70,6 +71,20 @@
             album: Object,
         },
         methods: {
+            selectRange(photo) {
+                var selection = this.$clipboard.getIds();
+                if (selection.length) {
+                    var lastAddedId = selection[selection.length - 1];
+                    var rangeStart = this.photos.findIndex((photo) => photo.getId() == lastAddedId);
+                    var rangeEnd = this.photos.indexOf(photo);
+                    var photosToBeAdded = this.photos.slice(Math.min(rangeStart, rangeEnd), Math.max(rangeStart, rangeEnd) + 1);
+                    for (var photo of photosToBeAdded) {
```

Comment:
Better use `let` instead of `var` and a different variable name here (photo is a function argument already).